### PR TITLE
Simplification of DMRGateway.ini

### PR DESCRIPTION
**Pull request by dj7lc**
This configuration assigns TS1 to DMRPlus/IPSC and TS2 to Brandmeister. DMRPlus Reflectors are on TS1 TG9. TS1 has to be empty in Brandmeister selfcare. In my opinion this is the simplest way use DMRGateway. Many configurations I encountered are using complex Talkgroup "NAT", so before I was able to make a contact I always had to search for a documentation of the one specific repeater I am using. In this configuration a single call to 5000 or the RPT-Command at 262994 tells me everything I need to know.

### CODE REVIEW

File: DMRGateway.ini [hunk 1]
```diff
@@ -48,49 +48,33 @@ Startup=950
...
-TGRewrite=2,9,2,9,1
-# Reflector control command slot 2 94000->4000 to 95000->5000
-PCRewrite=2,94000,2,4000,1001
-# Echo on RF slot 1 TG9990 to network slot 1 9990
-TypeRewrite=1,9990,1,9990
-SrcRewrite=1,9990,1,9990,1
-# Reflector status returns
-SrcRewrite=2,4000,2,9,1001
-# Pass all of the other private traffic on slot 1 and slot 2
+Enabled=1
```
Comment (phl0) [inline DMRGateway.ini]: Why did you delete the Name parameter from the config?

File: DMRGateway.ini [hunk 2]
```diff
@@ -48,49 +48,33 @@ Startup=950
...
-# Local=3352
-# Reflector TG on to slot 2 TG8
-TGRewrite=2,8,2,9,1
-# Echo on slot 2 TG9990
-TGRewrite=2,9990,2,9990,1
-# Reflector control command slot 2 84000->4000 to 85000->5000
-PCRewrite=2,84000,2,4000,1001
-# Pass all of the other talk group traffic on slot 1 and slot 2
-PassAllTG=1
+Enabled=1
```
Comment (phl0) [inline DMRGateway.ini]: Same here. Why is Name deleted?

**Comment by g4klx** [comment]
The included ini file is not meant to be used directly. It does not want to be simplified as it would not then show all of the available commands. Please leave it as it is.
